Phase2.1DataExtractionOracle.sql

```diff
@@ -23,6 +23,8 @@ set echo on ;
 --------------------------------------------------------------------------------
 -- General settings
 --------------------------------------------------------------------------------
+define start_date = '09/10/2021';
```

Comment:
Can you change it `define start_date=&1 -- '09/10/2021' `?

Reply:
Everything else looks good.

Reply:
Any reason for that? this date is a restriction from AKI that will not often get changed? this change will require us to enter it through Jenkins every time @lav-patel